### PR TITLE
LoginPage 

### PR DESCRIPTION
**Pull request by bholeneha**

### CODE REVIEW

File: app/api/auth/[...nextauth]/route.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	},
+	jwt: {
+		maxAge: 30
+	},
+	callbacks: {
+		// modify jwt info - add accesstoken to jwt payload 
+		jwt: async ({token, user}) => {
+			// console.log('jwt', token, user);
+			if (user) {
+				// @ts-ignore
```
Comment (np6176a) [inline app/api/auth/[...nextauth]/route.tsx:18]: When you use `ts-ignore` explain with a comment why.

File: app/components/atoms/button/button.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import './button.scss';
+
+type ButtonProps = {
+  onClick?: () => void;
+  children: React.ReactNode;
+  type?: any;
```
Comment (np6176a) [inline app/components/atoms/button/button.tsx]: Do not use `any`.
```suggestion
  type?: "button" | "submit" | "reset" | undefined;
```

File: app/components/atoms/button/button.stories.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+const meta: Meta<typeof Button> = {
+    title: 'Atoms/Button',
+    component: Button
+  };
+
+  export default meta;
+  
+  type Story = StoryObj<typeof Button>;
+
```
Comment (np6176a) [inline app/components/atoms/button/button.stories.ts:12]: Make sure all the props for the button shows up in the storybook docs. Currently `type` doesn't render. I believe you need to add it in the `args`.

File: app/components/molecules/navbar/navbar.stories.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+const meta: Meta<typeof NavBar> = {
+    title: 'Molecules/NavBar',
+    component: NavBar
+  };
+
+  export default meta;
+  
+  type Story = StoryObj<typeof NavBar>;
+  
+  export const Primary: Story = {
```
Comment (np6176a) [inline app/components/molecules/navbar/navbar.stories.ts:13]: The story is incomplete here. Is that on purpose?

File: app/components/molecules/navbar/navbar.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+const Navbar: React.FC<NavbarProps> = ({ navItems, currentActivePage }) => {
+
+  return (
+    <nav className='navbar'>
+      <ul>
+        {navItems.map((item) => (
+          // Check if the href of the item matches the currentActivePage
+          item.href === currentActivePage && (
```
Comment (np6176a) [inline app/components/molecules/navbar/navbar.tsx:22]: Considering we are using Next.js, how come we aren't using the Next.js native navigation: https://nextjs.org/docs/app/building-your-application/routing/linking-and-navigating

**Review by np6176a — COMMENTED** [review]
Overall, nice work. Good use of typescript. For the Form Field inputs, you can have your custom input using
```
<Field name="email">
                  {props => (
                      <div>
                          <label>Test</label>
                          <YourInputComponent {...props.input} />
                      </div>
                  )}
              </Field>
              ```
For the input component, you would need to extend the types to include `FieldRenderProps` One of the main reasons people use `interface Foo` over `type Foo = {}` syntax is that interface lets you extend. So in your case you would do something like `interface Input extends FieldRenderProps`.

A good start, but def needs some polish. Also the authentication bug is interesting, not sure what is happening in there and we may need to pair on it later, but for now let's ignore authentication and setup the routes properly with Next Js.

File: app/components/molecules/navbar/navbar.tsx (hunk 5 above)
Comment (bholeneha) [reply app/components/molecules/navbar/navbar.tsx:22]: I think I fixed this in the dashboard branch.

**Pull request merged**